queue.rb

```diff
@@ -0,0 +1,49 @@
+class Queue
+  attr_reader :size
+
+  def initialize
+    @queue = []
+    @size = 0
+  end
+
+  def enqueue(item)
+    @queue += [item]
+    @size += 1
+  end
+
+  def dequeue
+    dequeued = @queue[0]
+    @queue = @queue[1..(size-1)]
```

Comment:
I don't understand why size isn't an instance variable here. Also if what you're trying to do is go from the second element to the last element of the queue you just need to 
`@queue[1..-1]`  because -1 means the last item in the queue.